p2p_wallet/Resources/AppDelegate/AppDelegate+Injection.swift

```diff
@@ -277,6 +277,9 @@ extension Resolver: ResolverRegistering {
             .implements(JWTTokenValidator.self)
 
         register { Web3(rpcURL: String.secretConfig("ETH_RPC")!) }
+
+        register { LogoutServiceImpl() }
+            .implements(LogoutService.self)
```

Comment:
Should we use application scope here?

Reply:
The LogoutService carries BankTransferService, which is in .session scope:
<img width="592" alt="image" src="https://github.com/p2p-org/p2p-wallet-ios/assets/6975538/97002ab1-ae2a-4eb5-95ba-944bbd413995">
<img width="583" alt="image" src="https://github.com/p2p-org/p2p-wallet-ios/assets/6975538/e3ce5f5b-79f6-4b7c-a241-bea19639f38f">